xbmc/interfaces/json-rpc/VideoLibrary.cpp

```diff
@@ -910,7 +910,12 @@ JSONRPC_STATUS CVideoLibrary::Export(const std::string &method, ITransportLayer
 
 JSONRPC_STATUS CVideoLibrary::Clean(const std::string &method, ITransportLayer *transport, IClient *client, const CVariant &parameterObject, CVariant &result)
 {
-  std::string cmd = StringUtils::Format("cleanlibrary(video, %s)", parameterObject["showdialogs"].asBoolean() ? "true" : "false");
+  std::string cmd;
+  if (parameterObject["content"].empty())
+    cmd = StringUtils::Format("cleanlibrary(video, %s)", parameterObject["showdialogs"].asBoolean() ? "true" : "false");
```

Comment:
use new format string `{}` or positional `{0}`

Reply:
done. the new-fangled shit offers absolutely nothing here, but..

Reply:
it's slightly more performant because of how it's implemented. The new format strings are tested first and if no replacements are done it tries to fall back on old formats https://github.com/xbmc/xbmc/blob/master/xbmc/utils/StringUtils.h#L67